CollectionViewChallenge/CollectionViewChallenge/Views/FeedPreferencesPage.cs

```diff
@@ -0,0 +1,201 @@
+                        Visual = VisualMarker.Material,
+                        BorderColor = QantasCyan,
+                        CornerRadius = 2,
+                        Padding = 0,
+                        Content = new Grid
+                        {
+                            RowDefinitions =
+                            {
+                                new RowDefinition { Height = 115 },
+                                new RowDefinition { Height = 60 }
+                            },
+                            ColumnDefinitions =
+                            {
+                                new ColumnDefinition { },
+                                new ColumnDefinition { Width = 25 }
+                            },
+                            Children =
+                            {
+                                new Image
+                                {
+                                     BackgroundColor = QantasRed,
+                                     Aspect = Aspect.AspectFill
+                                }
+                                .Bind(Image.SourceProperty, nameof(FeedCategory.Image))
+                                .Row(0).Col(0).ColSpan(2),
+
+                                new Label
+                                {
+                                     Margin = new Thickness(10, 5),
+                                     FontFamily = RegularFont,
+                                     FontSize = 16,
+                                     TextColor = QantasText,
+                                }
+                                .Bind(Label.TextProperty, nameof(FeedCategory.Name))
+                                .Row(1),
+
+                                new Image
+                                {
+                                     Margin = new Thickness(0, 5, 0, 0),
+                                     Source = ImageSource.FromFile("ic_confirmed_16.png"),
+                                     HorizontalOptions = LayoutOptions.Start,
+                                     VerticalOptions = LayoutOptions.Start,
+                                     HeightRequest = 18,
+                                }
+                                .Bind(OpacityProperty, nameof(FeedCategory.Selected), converter: boolToOpacityConverter) 
+                                .Row(1).Col(1),
+                            }
+                        },
+                        Opacity = 0
+                    }.Invoke(x => x.FadeTo(1, 200)) // woof 
+                } 
+            };
+
+        private void SetupBindings() 
+        {
+            FeedCategories = 
+                Enumerable
+                    .Range(0, 10) // so we can test with more data 
+                    .SelectMany(_ => _categories.Select(FeedCategory.ForCategory)) 
+                    .ToObservableCollection();
+
+            CollectionView.SelectionChanged += (sender, e) =>
+            {
+                // this is not so great
```

Comment:
@rdavisau can you expand on this? What would you prefer/expect to have in order to manage selection?

Reply:
It was more a comment on the way my own code handling it - but the other ways it could work that I thought would be more natural from my perspective: 

* `ItemTapped` with args of `item`, then I can toggle its selected/unselected property
* `SelectionChanged` with args of `itemsAdded` and `itemsRemoved` so that I have the delta already

Either of those allow avoiding the full list iterations which is what I didn't like about my current code. I haven't checked how it's implemented in the renderers, so not sure whether we already do/don't have to do a full iteration internally to raise the event.  

Keep in mind I haven't thought about it as much as the team most likely has - I know I can get the outcome I want from the current api, and it's just as likely that that way it works now enables uses cases I wasn't thinking about.